build_dockers.yml

```diff
@@ -80,6 +80,14 @@
         - name: Set package subdir as fact
           set_fact: _package_subdir={{ _ls_register.stdout }}
 
+        - name: Build spark-base image locally
+          shell: >
+            docker build --no-cache  \
+                -t spark-base:latest  \
```

Comment:
Good idea. Used the specified tag in the base image and modified child image Dockerfiles on the fly using sed. PTAL.